src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -8,7 +8,6 @@
   <!--
     Set LanguageTargets to Microsoft.Common.targets for any project that the SDK won't (.proj, .noproj, etc)
     https://github.com/dotnet/sdk/blob/50ddfbb91be94d068514e8f4b0ce1052156364a0/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets#L28
-    
```

Comment:
Revert removed line breaks (?)

Reply:
Sorry, could not see it was whitespace removals...

Reply:
done